docs/release-notes/2.43.2.md

```diff
@@ -0,0 +1,23 @@
+# Percona Monitoring and Management 2.43.2
+
+| **Release date** | October 22nd, 2024                                                                                     |
+| -----------------| ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them, and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+##  PostgreSQL 17 support for Query Analytics (QAN)
+
+This patch release adds PostgreSQL 17 support for Query Analytics with updates to the `pg_stat_statements` and `pg_stat_monitor` extensions.
+
+To align with PostgreSQL 17 standards, we've renamed the `blk_read_time` field to `shared_blk_read_time` and the `blk_write_time` field to `shared_blk_write_time`.
```

Comment:
We also added `local_blk_read_time` and `local_blk_write_time` metrics, however we don't use them on dashboards yet, so probably we can omit them in RN.

Reply:
Thanks, Artem. I've confirmed with @BupycHuk  that we don't want to mention it yet.